packages/block-library/src/page-list/edit.js

```diff
@@ -133,6 +132,9 @@ export default function PageListEdit( {
 		renderAppender: false,
 		__unstableDisableDropZone: true,
 		templateLock: 'all',
+		onInput: NOOP,
+		onChange: NOOP,
+		value: blockList,
```

Comment:
This is what we were missing @draganescu

Reply:
So we don't use templates we just set the inner blocks. Hmm how to have thought of this :/

Reply:
> @talldan do we need both template lock and the onChange and onInput NOOPs?

It didn't seem to work without defining `onChange` / `onInput`. Possibly a bug, or possibly intentional. 🤔 

Template lock does make the fact that the block is not editable visibly noticeable, I wasn't sure if you wanted to keep it for that reason.